### ISSUE TITLE
Improve experience for Talkback users

### ISSUE THREAD

**Issue by cketti** [thunderbird/thunderbird-android#4794]
I just tried to use the app using [Talkback](https://support.google.com/accessibility/android/answer/6283677). The current experience is… not great.

There are a couple of places where controls are only read as "unlabeled button" or "unlabeled unchecked checkbox" etc.

Swipe to refresh isn't discoverable (see also [this forum post](https://forum.k9mail.app/t/change-request-please-add-an-manual-refresh-button-to-action-bar-or-more-menu/62)). I'd like to avoid having a "refresh" item in the overflow menu again. Maybe we can only include it when an accessibility service is enabled. Or maybe we can add an invisible control that will be picked up by Talkback.

I'm sure there are many more issues. Help by developers would be very much appreciated. Also feedback from experienced Talkback users.

**Comment by heull001** [comment]
I am not sure if I should post here, or open a new issue, so if a separate isue would be better I can open one.

In the messages list there are two elements for every message. First the message, followed by a button to star or unstar. So to search a message by swiping right or left, talkback users therefore needs twice as many actions as there are messages. One element per message would be better. The star/unstar button should be hidden for talkback users and added to the accessibility actions (<https://developer.android.com/guide/topics/ui/accessibility/principles?hl=en#accessibility-actions>). Actions which are only accessible via special guests like mark as read/unread or delete should be added to the a11y actions too.

**Comment by kewisch** [comment]
This is a very broad issue, I'd rather we focus on individual issues. What we could do eventually is a full accessibility audit and file individual issues from there that we could hand to users. @heull001 if you see ways to improve accessibility and would like to patch them please feel free to file an issue on what you would like to change (or a pull request if it is simple). If you just want to report the issue I'd say we wait until the audit, because it might not be very actionable short term.

Hope this works for you, let me know.
